### ISSUE TITLE
Documentation not rendered

### ISSUE THREAD

**Issue by ksurya** [pytorch/text#415]
<img width="984" alt="screen shot 2018-09-25 at 1 55 40 pm" src="https://user-images.githubusercontent.com/932927/46032877-c53f8000-c0ca-11e8-825b-214fe40ebb29.png">

**Comment by mttk** [comment]
Yup, there's some issues with rendering up-to-date documentation that we are aware of and working on fixing ASAP.
